gwos/contracts/always-success/Cargo.toml

```diff
@@ -6,4 +6,4 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-ckb-std = "0.9.0"
+ckb-std = "0.7.1"
```

Comment:
Why?

Reply:
Yes, it's rebase error. I have fixed it in a rebase commit.